modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -376,33 +380,113 @@ def _manage_skiprows_parameter(
         into integer value and `pre_reading` parameter will be set if needed
         (in this case fastpath can be done).
 
+        Additionally function is responsible for definition of number of rows that
+        should be skipped during data file partitioning (`skiprows_partitioning`
+        parameter) since it can depend on `skiprows` and `header` parameters
+        interconnections.
+
         Parameters
         ----------
         skiprows : int, array or callable, optional
-                Original skiprows parameter of read_csv function.
+            Original skiprows parameter of read_csv function.
         header_size : int, default: 0
-                Number of rows that are used by header.
+            Number of rows that are used by header.
 
         Returns
         -------
-        skiprows : int, array or callable
-                Updated skiprows parameter.
+        skiprows_md : int, array or callable
```

Comment:
What is `md`?

Reply:
`md` stands for Modin. In the context of this and `csv_dispatcher._read` functions `skiprows` is original `pandas.read_csv` `skiprows` parameter, while `skiprows_md` is parameter version modified for Modin specific purposes (sorting of list-like parameter, etc.)